migrations/202411120832_create_tables.sql

```diff
@@ -1,49 +1,34 @@
--- This database is designed to manage + store information for the dispatch system 
+-- This database is designed to manage and store information for the dispatch system
 CREATE DATABASE IF NOT EXISTS dispatch_system;
 
-USE dispatch_system;
```

Comment:
had to delete this because the migration script requires a table in the `mysql` db that keeps track of which migrations have been executed in order to see which new ones to apply.